README.md

```diff
@@ -76,6 +76,14 @@ You can install system-wide with:
 sudo make install
 ```
 
+To clean the directory of any build artifacts and return it to its initial checked out state, run:
```

Comment:
I’d suggest advising people to use a separate build directory instead. This is fairly broad advise for git itself

Reply:
I'm happy to make that change (especially for my own workflow!), but we should update the recommended build instructions in the `README.md` then, yeah?

We currently have:

```sh
cmake .
make
```

What should we change that to? @isaacbrodsky

Reply:
Any of these are roughly the same (slightly different if using cmake to start the build program vs. running `make` directly)

```
cmake -Bbuild
cmake --build build
```

or

```
cmake -Bbuild
cd build
make
```

or

```
mkdir build
cd build
cmake ..
make
```

Reply:
I like the last one!

Reply:
updated!